spec/predicates/vsa.md

```diff
@@ -1,11 +1,279 @@
+> used to perform verification.
+>
+> Each entry MUST contain a `digest` of the attestation and SHOULD contains a
+> `uri` that can be used to fetch the attestation.
+
+<a id="verificationResult"></a>
+`verificationResult` _string, required_
+
+> Either “PASSED” or “FAILED” to indicate if the artifact passed or failed the policy verification.
+
+<a id="verifiedProperties"></a>
+`verifiedProperties` _array (_string), required_
+
+> Indicates the properties verified for the artifact (and not
+> its dependencies), or "FAILED" if policy verification failed.
+>
+> If including SLSA levels, users MUST NOT include more than one level per SLSA
+> track. Note that each SLSA level implies all levels below it (e.g.
+> SLSA_BUILD_LEVEL_3 implies SLSA_BUILD_LEVEL_2 and SLSA_BUILD_LEVEL_1), so
+> there is no need to include more than one level per track.
+
+<a id="dependencyProperties"></a>
+`dependencyProperties` _object, optional_
+
+> A count of the dependencies verified to have each property.
+>
+> Map from _string to the number of the artifact's _transitive_ dependencies
+> that were verified at the indicated level. Absence of a given value
+>  MUST be interpreted as reporting _0_ dependencies with that value.
+>
+> If including SLSA levels, users MUST count each dependency only once per SLSA
+> track, at the highest level verified. For example, if a dependency meets
+> SLSA_BUILD_LEVEL_2, you include it with the count for SLSA_BUILD_LEVEL_2 but
+> not the count for SLSA_BUILD_LEVEL_1.
+
+<a id="slsaVersion"></a>
+`slsaVersion` _string, optional_
+
+> Indicates the version of the SLSA specification that the verifier used, in the
+> form `<MAJOR>.<MINOR>`. Example: `1.0`. If unset, the default is an
+> unspecified minor version of `1.x`.
+
+## Example
+
+WARNING: This is just for demonstration purposes.
+
+```jsonc
+"_type": "https://in-toto.io/Statement/v1",
+"subject": [{
+  "name": "out/example-1.2.3.tar.gz",
+  "digest": {"sha256": "5678..."}
+}],
+
+// Predicate
+"predicateType": "https://in-toto.io/attestation/verification_summary/v1",
+"predicate": {
+  "verifier": {
+    "id": "https://example.com/publication_verifier",
+    "version": {
+      "slsa-verifier-linux-amd64": "v2.3.0",
+      "slsa-framework/slsa-verifier/actions/installer": "v2.3.0"
+    }
+  },
+  "timeVerified": "1985-04-12T23:20:50.52Z",
+  "resourceUri": "https://example.com/example-1.2.3.tar.gz",
+  "policy": {
+    "uri": "https://example.com/example_tarball.policy",
+    "digest": {"sha256": "1234..."}
+  },
+  "inputAttestations": [
+    {
+      "uri": "https://example.com/provenances/example-1.2.3.tar.gz.intoto.jsonl",
+      "digest": {"sha256": "abcd..."}
+    }
+  ],
+  "verificationResult": "PASSED",
+  "verifiedProperties": ["SLSA_BUILD_LEVEL_3"],
+  "dependencyProperties": {
+    "SLSA_BUILD_LEVEL_3": 5,
+    "SLSA_BUILD_LEVEL_2": 7,
+    "SLSA_BUILD_LEVEL_1": 1,
+  },
+  "slsaVersion": "1.0"
+}
+```
+
+<div id="verificationresult">
+
+## Change history
```

Comment:
Include something about it moving here from slsa?

Reply:
Done